### ISSUE TITLE
ObjectPermission form field

### ISSUE THREAD

**Issue by eirsyl** [webkom/lego-webapp#1102]
Redux form field for fields in the soon to be ObjectPermissionSerializer...

**Comment by odinuge** [comment]
What is the best approach of these two:
```jsx
<ObjectPermissions
    requireAuth
    canViewGroups
    canEditUsers
    canEditGroups
/>
// can also be used like this:
<ObjectPermissions
    requireAuth
/>
<ObjectPermissions
    canViewGroups
    canEditUsers
    canEditGroups
/>
```

or
```jsx
<Filelds
   names={[
      'requireAuth',
      'canViewGroups',
      'canEditUsers',
      'canEditGroups'
    ]}
    component={ObjectPermissions.Field}
/>
// can also be used like this:
<Filelds
   names={[
      'requireAuth',
   ]}
   component={ObjectPermissions.Field}
/>
<Filelds
   names={[
      'canViewGroups',
      'canEditUsers',
      'canEditGroups'
   ]}
   component={ObjectPermissions.Field}
/>
```
The last one will require a bit more code and logic, but may be the most `redux-form` looking one. The last one will also require a normalization of the `requireAuth` fileld. We do need to map all the group and user selectors into arrays of `pk`s, instead of the searchmapping; so it more or less doesn't matter that much.

**Comment by odinuge** [comment]
Solved in #1104
